### ISSUE TITLE
How add attachments in the form of images to trello cards?

### ISSUE THREAD

**Issue by bnepryakhin63** [jchrom/trelloR#35]
Good afternoon! Tell me, is it possible to use your package to add attachments in the form of images to trello cards? Trying to find a solution, I tried to take a piece of your year and redo it by attachments, but instead of an image, a simple file comes in which a link to the directory where the file should originally lie. Can you tell me how to send an image?

`create_resource(
  resource = "card", path = "attachments", id = "5e*****e9afd",
  body = list(file = "C:/Users/***********/trello/sotrudniki.png",
              mimeType = "image/jpeg"
  ),
)`

**Comment by bnepryakhin63** [comment]
```
create_resource(
  resource = "card", path = "attachments", id = "5e*****e9afd",
  body = list(file = "C:/Users/***********/trello/sotrudniki.png",
              mimeType = "image/jpeg"
  ),
)
```

**Comment by jchrom** [comment]
Hello @bnepryakhin63

this is a bug. I have implemented a fix and pushed to the release candidate branch. You can install it like this:

`remotes::install_github("jchrom/trelloR@rc")`

After that, you can call a new wrapper to upload a file and use it as cover image:

`add_card_attachment(card_id, file = "~/Pictures/pic.jpg", cover = TRUE)`

This function uses the `httr::file_upload()` feature. So you could also call:

```
create_resource(
    resource = "card",
    path = "attachments",
    id = card_id,
    body = list(file = httr::upload_file("~/Pictures/pic.jpg"), setCover = TRUE)
  )
```

**Comment by bnepryakhin63** [comment]
thank you very much!

**Comment by jchrom** [comment]
This has now been released as part of 0.7.1 and can be installed from CRAN or the master branch in the usual way.
